requirements.txt

```diff
@@ -5,3 +5,4 @@ pre-commit~=3.6.2
 plotly~=5.22.0
 pandas~=2.1.3
 numpy~=1.26.2
+jinja~=3.1.4
```

Comment:
this creates an error for me
```
ERROR: Could not find a version that satisfies the requirement jinja~=3.1.4 (from versions: 0.8, 0.9, 1.0, 1.1, 1.2)
ERROR: No matching distribution found for jinja~=3.1.4
```
There is a package called `Jinja2` however. Same thing, different name?

Reply:
Yes, it need to be jinja2